payload/prepare_response.go

```diff
@@ -14,20 +15,34 @@ type PrepareResponse interface {
 	SetPreparationHash(h util.Uint256)
 }
 
-type prepareResponse struct {
-	preparationHash util.Uint256
-}
+type (
+	prepareResponse struct {
+		preparationHash util.Uint256
```

Comment:
Why not `PreparationHash` and drop `aux`?

Reply:
Because `prepareResponse` should implement `PrepareResponse` interface and this interface has `PreparationHash()` method.